app/webpacker/stylesheets/components/pages/home/_confidence.scss

```diff
@@ -75,8 +75,8 @@
         grid-column: 2;
         width: 87px;
         height: 53px;
-        justify-self: end;
         border: none;
+        float: right;
```

Comment:
If you make the class changes suggested in the other file, we can use the `justify-self: end;` rather than the float.

Reply:
Reverted back to justify-self after other changes.